CheatSheets/html-cheatsheet.md

```diff
@@ -25,100 +25,3 @@
-| `border=””` |Specifies border thickness of the image. If not mentioned, defaults to 0.|
-| `<map> … </map>` |Denotes an interactive (clickable) image.|
-| `<map name=””> … </map>` |Name of the map associated between the image and the map.|
-| `<area />` |Specifies image map area.|
-| `shape=””` |Shape of the area.|
-| `coords=””` | The coords attribute specifies the coordinates of an area in an image map. |
-
-**[🔼Back to Top](#table-of-contents)**
-
-## Lists
-
-| Command                | Description                                                                                                                               |
-| ---------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
-|`<ol> … </ol>`|Tag for ordered or numbered list of items.|
-|`<ul> … </ul>`|Contrary to the above tag, used for unorderedlist of items.|
-|`<li> … </li>`|Individual item as part of a list.|
-|`<dl> … </dl>`|Tag for list of items with definitions.|
-|`<dt> … </dt>`|The definition of a single term inline with body content.|
-|`<dd> … </dd>`|The description for the defined term.|
-
-**[🔼Back to Top](#table-of-contents)**
-
-## Forms
-
-| Command                | Description                                                                                                                               |
-| ---------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
-|`<form> … </form>`|The parent tag for an HTML form.|
-|`action=”url”`|The URL listed here is where the form data will be submitted once user fills it.|
-|`method=””`| It specifies which HTTP method (POST or GET) would be used to submit the form.|
-
-**[🔼Back to Top](#table-of-contents)**
-
-## Tables
-
-| Command                | Description                                                                                                                               |
-| ---------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- |
-|`<table> … </table>`|Marks a table in a webpage.|
-|`<caption> … </caption>`|Description of the table is placed inside this tag.|
-|`<thead> … </thead>`|Specifies information pertaining to specific columns of the table.|
-|`<tbody> … </tbody>`|The body of a table, where the data is held.|
-|`<tfoot> … </tfoot>`|Determines the footer of the table.|
-|`<tr> … </tr>`|Denotes a single row in a table.|
-|`<th> … </th>`|The value of a heading of a table’s column.|
-|`<td> … </td>`|A single cell of a table. Contains the actual value/data.|
-|`<colgroup> … </colgroup>`| Used for grouping columns together.
-|`<col>`|Denotes a column inside a table.|
-
-**[🔼Back to Top](#table-of-contents)**
```

Comment:
Please, delete your current `html-cheatsheet.md` file. Then, copy paste [html-cheatsheet.md](https://github.com/crescentpartha/CheatSheets-for-Developers/blob/main/CheatSheets/html-cheatsheet.md) file from the [Original Repo](https://github.com/crescentpartha/CheatSheets-for-Developers). Otherwise, It will be more complex to resolve.